### ISSUE TITLE
$article->tag('Gardening'); not work

### ISSUE THREAD

**Issue by 41410** [rtconner/laravel-tagging#77]
i find this example but not works.
$article->tag('Gardening'); // attach the tag

after i change it to array it works
$article->tag(array('Gardening'));

in tag() i found this code will remove the string 

		if(!is_array($tagNames)) {
			$tagNames = func_get_args();
			array_shift($tagNames);
		}

**Comment by rtconner** [comment]
I'm not able to reproduce this. What version of Laravel and what version of this plugin are you using?

**Comment by 41410** [comment]
"name": "laravel/framework",
            "version": "v5.1.19",

            "name": "rtconner/laravel-tagging",
            "version": "2.0.3",

**Comment by rtconner** [comment]
I tried it out on those versions. Can you make a gist or something with code to help me reproduce this?

**Comment by 41410** [comment]
remove the 
> array_shift($tagNames);
can fix it.

**Comment by rtconner** [comment]
Your avatar is hilarious.

**Comment by rtconner** [comment]
That function already exists and has for a while. Are you using the latest version of the plugin?
https://github.com/rtconner/laravel-tagging/blob/laravel-5/src/Taggable.php#L67

**Comment by 41410** [comment]
Yes. I am using the latest version.
But i still can use array for add tag anyway.

I found these codes added at Sep 13.
May I know what these codes for?

because i run these codes and i will get:

```php
$tagNames = 'iamtag';
tag($tagNames);
function tag($tagNames) {
    if(!is_array($tagNames)) {
        var_dump($tagNames); //string(6) "iamtag"
        $tagNames = func_get_args();
        var_dump($tagNames); //array(1) { [0]=> string(6) "iamtag" }
        array_shift($tagNames);
        var_dump($tagNames); //array(0) { }
    }
}
```

hope it can help you

**Comment by rtconner** [comment]
Another user point the issue out to me and was able to fix this in 2.0.5
